all.css

```diff
@@ -19,37 +29,443 @@ body {
   left: 52px;
   gap: 0px;
   opacity: 0px;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding: auto;
 }
-.navigation {
-  width: 401px;
-  height: 40px;
-  gap: var;
-  opacity: 0px;
+
+.navigation-list {
+  list-style-type: none;
+  display: flex;
 }
 
-.button {
-  text-decoration: none;
+.nav-link {
+  font-family: Poppins;
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 24px;
   color: #f5f8fb;
+  text-decoration-line: none;
+}
+
+.button-discord {
+  font-family: Poppins;
   font-size: 18px;
-  position: relative;
-  border-width: 2px;
-  border-style: solid;
-  border-color: #d0d7fb;
-  width: 188px;
-  height: 48px;
-  box-sizing: border-box;
-  font-family: "Poppins", serif;
   font-weight: 600;
-  font-style: normal;
+  line-height: 28px;
+  text-align: left;
+  text-decoration-line: none;
+  color: #f5f8fb;
 }
-.button--blue {
+.button-connect {
```

Comment:
The following buttons have the same styles. Please create and use one class for them, for example 'button-primary'
<img width="85" alt="Screenshot 2024-11-26 at 11 04 34 PM" src="https://github.com/user-attachments/assets/95a74687-a7e4-4612-b701-3deaba9ca799">
<img width="99" alt="Screenshot 2024-11-26 at 11 04 41 PM" src="https://github.com/user-attachments/assets/e961ac2b-92ba-4aa3-a4fc-e6268dcfd357">
<img width="33" alt="Screenshot 2024-11-26 at 11 04 47 PM" src="https://github.com/user-attachments/assets/42c22d6f-2736-4312-b46d-0c537aa0845d">

Reply:
done